fetch.bs

```diff
@@ -3684,11 +3785,35 @@ steps:
 
  <li><p>Wait for either <var>response</var>'s <a for=response>body</a> to be null, or
  <var>response</var>'s <a for=response>body</a>'s <a for=body>stream</a> to be
- <a for=ReadableStream>closed</a> or <a for=ReadableStream>errored</a>, and then set
- <var>request</var>'s <a>done flag</a>.
- <!-- This is really bad and needs to be handled differently at some point. -->
+ <a for=ReadableStream>closed</a> or <a for=ReadableStream>errored</a>, and perform the following steps:
+ <ol>
+  <li><p>Set <var>request</var>'s <a>done flag</a>.
+   <!-- This is really bad and needs to be handled differently at some point. -->
+  <li>Call <a for=/>report resource timing</a> with <var>fetchParams</var> and <var>response</var>.
+ </ol>
 </ol>
 
+<p>To <dfn>report resource timing</dfn>, given <a for=/>fetch params</a> <var>fetchParams</var> and
```

Comment:
The normalization would happen here, and then it would call an algo in RT with the normalized result. I need to have this patch in first, as the algo in RT would need to be aware of `fetch timing info` and `connection timing info` to receive them as parameters.